core_data_modules/traced_data/io.py

```diff
@@ -0,0 +1,127 @@
+import time
+
+import six
+import unicodecsv
+from core_data_modules import Metadata
+
+from core_data_modules.util import SHAUtils, TextUtils
+
+
+class TracedDataCodaIO(object):
+    @staticmethod
+    def export_coda(data, key, f):
```

Comment:
@as2388 could we rename this export_x_to_y to be more explicit. The type system isn't much help to us here.